src/input/type_record.cc

```diff
@@ -483,13 +485,17 @@ AbstractRecord::AbstractRecord()
 
 AbstractRecord::AbstractRecord(const AbstractRecord& other)
 : child_data_(other.child_data_)
-{}
+{
+	attributes_ = other.attributes_;
```

Comment:
Rather call TypeBase copy constructor in initializer list. Need also add to other copy constructors, are there any for Record or Selection?